src/Blocks/BlockStairs.h

```diff
@@ -115,6 +115,43 @@ class cBlockStairsHandler :
 			}
 		}
 	}
+
+	virtual double GetBlockHeight(void) override
+	{
+		return 1.0;
+	}
+
+	/** EXCEPTION a.k.a. why is this removed:
+		This collision-detection is actually more accurate than the client, but since the client itself
+		sends inaccurate / sparse data, it's easier to just err on the side of the client and keep the
+		two in sync by assuming that if a player hit ANY of the stair's bounding cube, it counts as the ground. */
+#if 0
```

Comment:
Style:
- comments should all start at the same indent level
- preprocessor directives should be indented the same as the code around them.